src/org/parosproxy/paros/extension/option/OptionsCertificatePanel.java

```diff
@@ -991,10 +999,21 @@ private OptionsCertificatePanel getContentPane() {
 	public void initParam(Object obj) {
 		OptionsParam options = (OptionsParam) obj;
 		OptionsParamCertificate certParam = options.getCertificateParam();
-		useClientCertificateCheckBox.setSelected(certParam.isUseClientCert());
+		
+		// Should only run once after startup if client certificate is set from commandline
+		if(overrideEnableClientCertificate) {
+			certParam.setEnableCertificate(true);

```

Comment:
We should not need to call this, the `OptionsParamCertificate` already enables the certificate.

Reply:
From talks in IRC, this does need to be called (for when persist is false).